source/compat.rst

```diff
@@ -90,3 +90,25 @@ best results, one of the following browsers recommended:
 -  Google Chrome version 36.0+
 
 -  Safari 6+
+
+Notice Of Management OSes and Hypervisors to be Deprecated
+----------------------------------------------------------
+
+The following hypervisors will no longer be tested from the first release AFTER
+1st June 2018. And will be removed from the CloudStack codebase from the first 
```

Comment:
Do we give any warranty about a number of releases *between* June 1st and Novermber 1st?!!!

Reply:
No, there is no real way to know what versions will be out in that time right now.

Reply:
of course can we add that remark? something like "There is no way of telling if there will be a version out between those two dates."